src/client/main.cpp

```diff
@@ -62,6 +63,11 @@ static void _libhook_init() {
       void *, uint8_t *, int32_t, int32_t, int32_t, int, void (*)(void *),
       void *))dlsym(RTLD_NEXT, "_ZN6QImageC1EPhiiiNS_6FormatEPFvPvES2_");
 
+  if (getenv("RM2FB_DISABLE") != "") {
```

Comment:
`getenv` returns a `char*`, which can't be compared to string literals like this.
If there is no env. variable with the given name it will return `null`, so maybe just check against that?

Reply:
ooof. there's another place i made that mistake, thanks for catching!